### ISSUE TITLE
Dead links: festvox.org doesn't work anymore?

### ISSUE THREAD

**Issue by CryptoCashCashoo** [MycroftAI/mimic#225]
**Describe the bug**
The links to festvox.org doesnt work. Is the site permanently down?

**Comment by forslund** [comment]
Will need to research but you are right, it's not responding. Maybe we should link to the https://github.com/festvox/ github project?

**Comment by krisgesling** [comment]
Yeah, I couldn't immediately see where pre-compiled voices exist on their github though. Did they have more than what is included in `Mimic1/voices/` already?

**Comment by ddavout** [comment]
> Its back now, the machine died after only 16 years service, and its not
> restored, there might still be issues, but it seems to be mostly working

https://github.com/festvox/festvox/issues/30#issuecomment-1274206581
